assets/prometheus-k8s/prometheus.yaml

```diff
@@ -197,7 +197,8 @@ spec:
       openshift.io/cluster-monitoring: "true"
   ruleSelector: {}
   scrapeClasses:
-  - name: tls-client-certificate-auth
+  - fallbackScrapeProtocol: PrometheusText0.0.4
```

Comment:
Let's just have it for UWM as we don't have control over its targets.
targets scraped by Platform Prom should be fixed instead.